test/forms/memberDetails/MemberNameFormProviderSpec.scala

```diff
@@ -94,4 +95,23 @@ class MemberNameFormProviderSpec extends StringFieldBehaviours {
       requiredError = FormError(fieldName, requiredKey)
     )
   }
+
+  "MemberNameFormProvider" - {
+
+    "must allow leading and trailing spaces and trim them on binding" in {
+      val result = form.bind(
+        Map(
+          "memberFirstName" -> "  Jane  ",
+          "memberLastName"  -> "  Doe  "
+        )
+      )
+
+      result.errors mustBe empty
+
+      val PersonName(firstName, lastName) = result.value.value
+
+      firstName mustBe "Jane"
+      lastName mustBe "Doe"
+    }
```

Comment:
Can we add a test for mulitple first name and last name. "Shelley Ann", "van Basten"